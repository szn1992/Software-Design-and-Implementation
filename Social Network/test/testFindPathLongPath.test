#Test the output of findPath in a complicated graph

# Load the graph
LoadGraph marvel marvel.tsv

# Find the path
FindPath marvel WARLORD WU,_LEIKO

FindPath marvel LUPO ROM_CLONE

FindPath marvel OSAMA ROM_CLONE