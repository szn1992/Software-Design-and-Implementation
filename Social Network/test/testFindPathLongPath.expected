#Test the output of findPath in a complicated graph

# Load the graph
loaded graph marvel

# Find the path
path from WARLORD to WU, LEIKO:
WARLORD to DAGGER/TANDY BOWEN via AA2 18
DAGGER/TANDY BOWEN to ANT-MAN/DR. HENRY J. via FF@ 22
ANT-MAN/DR. HENRY J. to WU, LEIKO via H2 434

path from LUPO to ROM CLONE:
LUPO to ANGEL/WARREN KENNETH via M/FAN 3
ANGEL/WARREN KENNETH to DARKSTAR/LAYNIA SERG via CHAMP 10
DARKSTAR/LAYNIA SERG to ROM CLONE via ROM 45

path from OSAMA to ROM CLONE:
OSAMA to AURORA/JEANNE-MARIE  via UX 120
AURORA/JEANNE-MARIE  to DARKSTAR/LAYNIA SERG via COC 1
DARKSTAR/LAYNIA SERG to ROM CLONE via ROM 45
